optimum/habana/transformers/generation/stopping_criteria.py

```diff
@@ -44,7 +44,3 @@ def gaudi_MaxNewTokensCriteria_call(self, input_ids: torch.LongTensor, scores: t
         return token_idx >= self.max_length
     else:
         return input_ids.shape[-1] >= self.max_length
-
-
-def gaudi_StoppingCriteriaList_call(self, input_ids: torch.LongTensor, scores: torch.FloatTensor, **kwargs) -> bool:
-    return any(criteria(input_ids, scores, **kwargs) for criteria in self)
```

Comment:
@regisss without this, would the previous issue fixed? https://github.com/huggingface/optimum-habana/pull/627

Reply:
Yes, the goal of `gaudi_StoppingCriteriaList_call` was to add the `**kwargs` that were missing in Transformers v4.37 but I added them on time for Transformers v4.38: https://github.com/huggingface/transformers/blob/v4.38.2/src/transformers/generation/stopping_criteria.py#L132